app/views/meeting.details.html

```diff
@@ -1,119 +1,93 @@
 
             <!-- PRIMARY BLOCK HEADER -->
-            <div class="btn-group db-block-hdr db-prim-bg db-brd-top db-padding-2">
-
-                <button ng-if="menuClosed()" type="button" class="btn db-btn-prim db-btn-small db-brd db-margin-2" ng-click="openMenu()"><span class="glyphicon glyphicon-menu-hamburger"></span></button>
-                <button ng-if="menuClosed()" type="button" class="btn db-btn-prim db-btn-small db-brd db-margin-2" ng-click="c.toggleParallelMode()">
+            <div class="btn-group db-flex-wrap db-prim-bg db-brd-top db-padding-2">
+                <button ng-if="menuClosed()" type="button" class="btn db-btn-prim db-btn-small db-brd-all db-margin-2" ng-click="openMenu()">
+                    <span class="glyphicon glyphicon-menu-hamburger"></span>
+                </button>
+                <button ng-if="menuClosed()" type="button" class="btn db-btn-prim db-btn-small db-brd-all db-margin-2" ng-click="c.toggleParallelMode()">
                     <span class="glyphicon ng-class:[{'glyphicon-resize-vertical':parallelMode},{'glyphicon-resize-horizontal':!parallelMode}]"></span>
                 </button>
-                <button type="button" class="btn db-btn-prim db-btn-small db-brd db-margin-2" ng-disabled="parallelMode" ng-click="c.primaryClicked()">
+                <button type="button" class="btn db-btn-prim db-btn-small db-brd-all db-margin-2" ng-disabled="parallelMode" ng-click="c.primaryClicked()">
                     <span class="glyphicon ng-class:[{'glyphicon-minus':c.bm === c.bms.PRIMARY},{'glyphicon-plus':c.bm !== c.bms.PRIMARY}]"></span>
                 </button>
-
+                <button type="button" class="btn db-btn-sel db-btn-big db-brd-all db-margin-2 active">
+                    <span translate translate-cloak>STR_PRESENTATION</span>
+                </button>
             </div>
 
             <!-- PRIMARY BLOCK CONTENT -->
             <div class="db-block-content db-block-border db-brd-bottom">
-
-                <db-pdf ng-if="!c.isSecret(c.tData)" ng-attr-uri="c.primaryUrl"></db-pdf>
-                <db-imgviewer ng-if="c.isSecret(c.tData)" file-conf="c.tData"></db-imgviewer>
+                <db-pdf ng-if="c.pm === c.pms.DEFAULT" ng-attr-uri="c.primaryUrl"></db-pdf>
+                <db-imgviewer ng-if="c.pm === c.pms.SECRET" file-conf="c.tData"></db-imgviewer>
                 <div ng-if="c.error" class="alert alert-danger" role="alert">{{c.error}}</div>
-
             </div>
 
         </div>
         <!-- PRIMARY BLOCK END -->
 
-        <!-- SECONDARY  BLOCK START -->
-        <div class="db-block db-margin-05 ng-class:[{'db-block-close':c.bm === c.bms.PRIMARY},{'db-block-open':c.bm !== c.bms.PRIMARY}]">
+        <!-- SECONDARY BLOCK START -->
+        <div class="db-block ng-class:[{'db-block-close db-brd-all':c.bm === c.bms.PRIMARY},{'db-block-open':c.bm !== c.bms.PRIMARY},{'db-margin-05':!isMobile}]">
```

Comment:
About isMobile,
 `meeting.details.ctrl.js` takes a copy of `$rootScope.isMobile` and uses that while view template uses rootScope.isMobile. 
There's no evident bug, but should they use the same copy? If rootScope.isMobile changes it will be now reflected on view template but not in controller logic.